src/components/header/Header.jsx

```diff
@@ -25,54 +29,58 @@ import useGetIOUKeys from '../../hooks/useGetIOUKeys'
 import Dropdown from 'react-dropdown';
 import 'react-dropdown/style.css';
 import { useCookies } from 'react-cookie';
-import * as a from '../../api/chain'
+import * as a from '../../api/chain';
+
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import MenuIcon from "@material-ui/icons/Menu";
+
 
 const LinkBehavior = forwardRef((props, ref) => (
   <RouterLink ref={ref} to={ROUTES.main} {...props} />
 ));
 
-const Header = ({ classes }) => {
+const Header = ({ classes, setMenuVisibility }) => {
   const [isNotifications, setIsNotifications] = useState(false);
   const [isFavorite, setIsFavorite] = useState(false);
   const dataIOUsList = useGetIOUs()
   const dataIssuers = useGetIssuers()
   const dataIOUKeys = useGetIOUKeys()
-  const dappChains = require("../../assets/dappChains.json") 
+  const dappChains = require("../../assets/dappChains.json")
```

Comment:
Why are we using dynamic import with `require` here?
Can we use the static import above like:
`import dappChains from "../../assets/dappChains.json"`